router/dataplane_spec.gobra

```diff
@@ -18,13 +18,26 @@ package router
 
 import (
 	"net"
+	"hash"
+	"github.com/scionproto/scion/verification/utils/slices"
+	"github.com/scionproto/scion/verification/utils/definitions"
```

Comment:
let's use `sl` and `def` as the qualifiers for `slices` and `definitions` respectively